web/src/containers/QuarterlyBudgetSummary.js

```diff
@@ -7,150 +7,130 @@ import { t } from '../i18n';
+                    className="right-align"
+                    id={`quarterly-budget-summary-${source}-fy-${year}-subtotal`}
+                  >
+                    {t('table.subtotal')}
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                {Object.keys(EXPENSE_NAME_DISPLAY).map(name => (
+                  <tr
+                    key={name}
+                  >
                     <th
-                      className="bg-gray-light"
-                      id={`quarterly-budget-summary-${source}-total2`}
-                    />
-                  </tr>
-                </thead>
-                <tbody>
-                  {Object.keys(EXPENSE_NAME_DISPLAY).map(name => (
-                    <tr
-                      key={name}
-                      className={`${name === 'combined' ? 'bold' : ''}`}
+                      headers={`quarterly-budget-summary-${source}-null1 quarterly-budget-summary-${source}-null2`}
                     >
+                      {EXPENSE_NAME_DISPLAY[name]}
+                    </th>
+                    {QUARTERS.map(q => (
                       <td
-                        headers={`quarterly-budget-summary-${source}-null1 quarterly-budget-summary-${source}-null2`}
-                      >
-                        {EXPENSE_NAME_DISPLAY[name]}
-                      </td>
-                      {years.map((year, i) => (
-                        <Fragment key={year}>
-                          {QUARTERS.map(q => (
-                            <td
-                              className={`mono right-align nowrap ${
-                                name === 'combined' ? `${color(i)}-light` : ''
-                              }`}
-                              key={q}
-                              headers={`quarterly-budget-summary-${source}-fy-${year} quarterly-budget-summary-${source}-fy-${year}-q${q}`}
-                            >
-                              <Dollars>{data[year][q][name]}</Dollars>
-                            </td>
-                          ))}
-                          <td
-                            className={`bold mono right-align nowrap ${color(
-                              i
-                            )}-light`}
-                            headers={`quarterly-budget-summary-${source}-fy-${year} quarterly-budget-summary-${source}-fy-${year}-subtotal`}
-                          >
-                            <Dollars>{data[year].subtotal[name]}</Dollars>
-                          </td>
-                        </Fragment>
-                      ))}
-                      <td
-                        className="bold mono right-align nowrap bg-gray-light"
-                        headers={`quarterly-budget-summary-${source}-total2 quarterly-budget-summary-${source}-total`}
+                        className={`mono right-align nowrap ${
+                          name === 'combined'
+                        }`}
```

Comment:
Two questions here.  One, is there a plan to come back and change up the classes on this one?  Looks like it's still using the old atomic styles.

Two, it looks like the class will either be `mono right-align nowrap` if the name is "combined" or else `mono right-align nowrap false`?  Is that intended?

![scratch chin](https://media.giphy.com/media/l0HlOqrRaehu4Iqly/giphy.gif)

Reply:
Yeah, I figured we'd clean up those classes when we know for sure what we want the tables to look like. Good catch on the `combined` one -- that was me apparently not cleaning up _all_ of a ternary operator. 😬